cypress/integration/unit/settings/general/shouldTestTheModalBehindManageGovermentDomains.spec.js

```diff
@@ -7,29 +7,35 @@ import modal from "../../../../selectors/modal.selectors";
 import form from "../../../../selectors/form.selectors";
 
 context('Settings page tests', () => {
-
   let govermentDomains = [];
-  govermentDomains.push('Cultuur, jeugd, sport & media',
-    'Economie, wetenschap & innovatie',
-    'Financieën & begroting',
-    'Internationaal Vlaanderen',
-    'Kanselarij & bestuur',
-    'Landbouw & visserij',
-    'Mobiliteit & openbare werken',
-    'Omgeving',
-    'Onderwijs & vorming',
-    'Welzijn, vollksgezondheid & gezin',
-    'Werk & sociale economie'
-  );
+
+  function insertData(){
+    govermentDomains.push('Cultuur, jeugd, sport & media',
+      'Economie, wetenschap & innovatie',
```

Comment:
Ik zou hier de array teruggeven zodat je ze in de beforeEach kunt steken in de variabele ipv een bestaande variabele op te vullen, dan is de methode een pure function.